### DISCUSSION TITLE
How do i change my name

### DISCUSSION THREAD

**Discussion by sagwamatrlawns** [community/community#18776] [Other Feature Feedback, Questions, & Ideas]
How do i change my name

**Comment by Zi-Tao** [comment]
@sagwamatrlawns https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username

**Comment by TenTraicion** [comment]
Go to: Setting> Public Profile> Name to change Account Name
and Setting> Account> Change Username to change the username of your github account.

**Comment by diegoleondev** [comment]
![Frame 1](https://user-images.githubusercontent.com/76979144/177057394-17f64d9c-4e2b-4dfc-909b-8609c7cf12dd.png)

**Comment by 0xdsm** [comment]
Hello @sagwamatrlawns! Hope you are well

You can follow the official GitHub documentation to see more details and better instructions: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username

Or you can simple follow this workflow: Click on your profile picture on top bar > Click in the Settings option > Go to Account > Click in Change Username Button

Best regards,
oppsec.
